lib/inferno/dsl/fhir_evaluator/data_summary.rb

```diff
@@ -0,0 +1,96 @@
+module FhirEvaluator
```

Comment:
Anything inside Inferno needs to be inside the `Inferno` module. Anything in `inferno/dsl` needs to be inside `Inferno::DSL`.

Reply:
The file is deleted.